core/src/test/java/io/aiven/klaw/config/ManageDatabaseTest.java

```diff
@@ -25,9 +26,13 @@ public class ManageDatabaseTest {
   private ManageDatabase manageDatabase = new ManageDatabase();
   @Mock private HandleDbRequestsJdbc handleDbRequests;
 
+  @Mock HighAvailabilityUtilsService highAvailabilityUtilsService;
+
   @BeforeEach
   public void setUp() {
     ReflectionTestUtils.setField(manageDatabase, "handleDbRequests", handleDbRequests);
+    ReflectionTestUtils.setField(
+        manageDatabase, "highAvailabilityUtilsService", highAvailabilityUtilsService);
```

Comment:
May be this is not required if this unused field is removed from ManageDatabase ?